spec/system/admin/order_spec.rb

```diff
@@ -719,7 +719,7 @@ def new_order_with_distribution(distributor, order_cycle)
             it "should not display links but a js alert" do
               visit spree.edit_admin_order_path(order)
 
-              find("#links-dropdown .ofn-drop-down").click
+              find("#links-dropdown .ofn-drop-down details").click
```

Comment:
I like to set these up from the user's perspective, so would have tried something like this (untested):
```
find("summary", text: "Actions").click
```

But this way is fine too. It may need adjusting after addressing my comment above though.

Reply:
I like this version, I updated in that way.